sdk/core/Azure.Core.TestFramework/src/TestLogger.cs

```diff
@@ -27,44 +27,14 @@ internal class TestLogger : IDisposable
         /// </summary>
         private AzureEventSourceListener Listener { get; }
 
-        /// <summary>
-        /// A buffer containing the logs collected thus far.  It is initialized
-        /// in SetupEventsForTest and (optionally) written out in
-        /// OutputEventsForTest.
-        /// </summary>
-        private StringBuilder _log;
-
-        /// <summary>
-        /// Gets a value indicating whether or not to log output for successful tests.
-        /// This could be treated as a numeric level in the future if we wanted different
-        /// levels of verbosity, but for now is only boolean.
-        /// </summary>
-        private readonly bool _verbose = Environment.GetEnvironmentVariable("AZURE_SEARCH_TEST_VERBOSE") != null;
```

Comment:
I think @heaths added this variable to some pipeline somewhere, we need to remove it.

Reply:
It's in sdk/seach/tests.yml in the `EnvVars` template parameter.